checker/requestcache_test.go

```diff
@@ -129,12 +129,12 @@ func TestInitPeerBothLabels(t *testing.T) {
 	}
 	uut, err := NewRequestCache(store, req)
 	Expect(err).To(Succeed())
-	Expect(uut.Source().Name).To(Equal("bacon"))
-	Expect(uut.Source().Namespace).To(Equal("sandwich"))
-	Expect(uut.Source().Labels).To(Equal(map[string]string{"k1": "v1", "k2": "v2", "k5": "v5", "k6": "v6"}))
-	Expect(uut.Destination().Name).To(Equal("ham"))
-	Expect(uut.Destination().Namespace).To(Equal("sub"))
-	Expect(uut.Destination().Labels).To(Equal(map[string]string{"k3": "v3", "k4": "v4", "k7": "v7", "k8": "v8"}))
+	Expect(uut.SourcePeer().Name).To(Equal("bacon"))
+	Expect(uut.SourcePeer().Namespace).To(Equal("sandwich"))
+	Expect(uut.SourcePeer().Labels).To(Equal(map[string]string{"k1": "v1", "k2": "v2", "k5": "v5", "k6": "v6"}))
+	Expect(uut.DestinationPeer().Name).To(Equal("ham"))
+	Expect(uut.DestinationPeer().Namespace).To(Equal("sub"))
+	Expect(uut.DestinationPeer().Labels).To(Equal(map[string]string{"k3": "v3", "k4": "v4", "k7": "v7", "k8": "v8"}))
 }
```

Comment:
Do we need tests for `Source|DestinationNamespace`

Reply:
Good catch!